packages/database/src/migrations/20210630053253-AddPostcodePercentFeverOverlayFlutracking-modifies-data.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const MAP_OVERLAY = {
+  id: 'AU_Flutracking_Postcode_Fever_Percent',
+  name: '% fever and cough by postcode',
+  userGroup: 'Public',
+  dataElementCode: 'value',
+  isDataRegional: true,
+  measureBuilder: 'useReportServer',
+  measureBuilderConfig: {
+    dataSourceType: 'custom',
+    reportCode: 'AU_Flutracking_Postcode_Percent_Report',
+  },
+  presentationOptions: {
```

Comment:
@rohan-bes  displayOnLevel: 'District' was removed as performance is looking _slightly_ better. Can add back if needed.